nativeunwind/elfunwindinfo/elfgopclntab.go

```diff
@@ -498,6 +499,17 @@ func (g *Gopclntab) mapPcval(offs int32, startPc, pc uint) (int32, bool) {
 	return p.val, true
 }
 
+func (g *Gopclntab) LookupFunction(funcName string) (uintptr, error) {
```

Comment:
Looks like a duplicate to `PCForSymbol()` in https://github.com/open-telemetry/opentelemetry-ebpf-profiler/pull/728

Reply:
Sorry about that, I did not see you already opened a PR for this. I will close this one...

Reply:
no worry :) it was just pushed earlier this morning.